README.md

```diff
@@ -7,14 +7,10 @@
 # To build this application:
 
 ```
-$ mkdir wgo
-$ cd wgo
-$ export GOPATH=`pwd`
-$ export GOBIN=$GOPATH/bin
-$ export PATH=$GOBIN:$PATH
 $ go get github.com/skelterjohn/wgo
 $ go install github.com/skelterjohn/wgo
```

Comment:
Why do we need `go get` followed by `go install`?
According to https://golang.org/cmd/go/, a `go get` also performs an install.